packages/medusa/src/loaders/api.ts

```diff
@@ -27,5 +31,18 @@ export default async ({ app, container, configModule }: Options) => {
   app.use(bodyParser.json())
   app.use("/", routes(container, configModule.projectConfig))
 
+  try {
+    /**
+     * Register the Medusa CORE API routes using the file based routing.
+     */
+    await new RoutesLoader({
+      app,
+      rootDir: path.join(__dirname, "../api/routes"),
+      configModule,
+    }).load()
+  } catch (err) {
+    logger.warn(`An error occurred while registering Medusa Core API Routes`)
```

Comment:
thought(non-blocking): Should maybe throw here since this is core routes? Plugin routes are less critical and we can live with those not registering properly. What do you think?

suggestion(non-blocking): Log the error stack trace, if it's worth anything

Reply:
Good point